backend/src/resolvers/shout.js

```diff
@@ -0,0 +1,51 @@
+export default {
+  Mutation: {
+    shout: async (_object, params, context, _resolveInfo) => {
+      const { id, type } = params
+
+      const session = context.driver.session()
+      let transactionRes = await session.run(
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-04-11T22:47:32Z" title="Friday, April 12th 2019, 12:47:32 am +02:00">Apr 12, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,51 @@
+export default {
+  Mutation: {
+    shout: async (_object, params, context, _resolveInfo) => {
+      const { id, type } = params
+
+      const session = context.driver.session()
+      let sessionRes = await session.run(
```


I don't love the name `sessionRes`
I feel like we should be naming it what is the actual result, which would tempt me to name it `isShouted`, but then I see on line 19, we seem to get a list of records back that we then need to grab `isShouted` from.

I was having a read here https://neo4j.com/docs/driver-manual/current/sessions-transactions/, and it seems the result is from the transaction, not the session. Maybe it would be better `transactionRes`, but maybe I'm just being pedantic, and can be ignored.

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-04-12T06:36:26Z" title="Friday, April 12th 2019, 8:36:26 am +02:00">Apr 12, 2019</time>_

---


I am really happy to rename `sessionRes`  to ´transactionRes´ @mattwr18 . That is a very good clarification we can use in the future for all customised mutations. 🤗

I am happy especially if there is nothing else to mention from your side. 😁